aws/validators.go

```diff
@@ -1614,6 +1615,18 @@ func validateWafMetricName(v interface{}, k string) (ws []string, errors []error
 	return
 }
 
+func validateWafPredicatesType() schema.SchemaValidateFunc {
+	return validation.StringInSlice([]string{
+		waf.PredicateTypeIpmatch,
+		waf.PredicateTypeByteMatch,
+		waf.PredicateTypeSqlInjectionMatch,
+		waf.PredicateTypeGeoMatch,
```

Comment:
`waf.PredicateTypeGeoMatch` and `waf.PredicateTypeRegexMatch` where added here, do we actually support them (e.g. they do not require additional resource code/attributes)?